zokrates_fs_resolver/src/lib.rs

```diff
@@ -20,11 +20,13 @@ fn resolve_with_location(
     location: &String,
     source: &String,
 ) -> Result<(BufReader<File>, String, String), io::Error> {
-    let source = PathBuf::from(source);
+    let mut source = PathBuf::from(source);
+
+    source.set_extension("code");
```

Comment:
I think this should happen earlier, where the resolver is called: the fact that we interpret `foo` as `foo.code` is not specific to `fs_resolver` so the `source` argument should already have the extension here. The resolver only fetches the file.

Reply:
I moved the extension addition to `zokrates_core/src/imports.rs`, much nicer this way. Also restored the tests in the `fs_resolver` which is a lot cleaner. 
Two remaining questions:
1. For the imports of github files, do we want to keep the extensions there or rather not? Currently they are not needed, so the format is the same as importing a local file
2. Tests are now not working because the `github_resolver` test gets its testing file from the master branch, which still contains the file extension. Should I leave it like this or change the imports, so the file extension can be passed, but doesn't have to.